internal/v2/s2av2.go

```diff
@@ -6,14 +6,25 @@ import (
 	"errors"
```

Comment:
For this PR, I tested it by starting up the client, server, and fakes2av2 server binaries, I'll add screenshots(with and without the logging) in the description. I agree that it would be a good idea to add unit tests to test the Client and Server Handshakes in the success case; is it ok if I add this as a TODO and address in a follow up PR?

Reply:
For this PR, I tested it by starting up the client, server, and fakes2av2 server binaries, I'll add screenshots(with and without the logging) in the description. I agree that it would be a good idea to add unit tests to test the Client and Server Handshakes in the success case; is it ok if I add this as a TODO and address in a follow up PR?